tests/components/prowl/test_config_flow.py

```diff
@@ -0,0 +1,111 @@
+"""Test Prowl config flow."""
+
+import pytest
+
+from homeassistant import config_entries
+from homeassistant.components.prowl.const import DOMAIN
+from homeassistant.const import CONF_NAME
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+
+from .conftest import (
+    BAD_API_RESPONSE,
+    CONF_INPUT,
+    CONF_INPUT_NEW_KEY,
+    INVALID_API_KEY_ERROR,
+    OTHER_API_KEY,
+    TIMEOUT_ERROR,
+)
+
+
+@pytest.mark.asyncio
```

Comment:
We don't need this, we have configured pytest for asyncio. Please remove it from all tests.